roles/install_ansible_rulebook/tasks/install_darwin.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Ensure system dependencies
+  ansible.builtin.package:
+    name:
+      - git
+      - "python@3.10"
+      - "openjdk@17"
+      - maven
+      - gcc
+
+- shell: brew --prefix openjdk@17
+  register: brew_jdk
+
+- name: Creating a symlink
+  ansible.builtin.file:
+    src: "{{ brew_jdk.stdout_lines[0] }}/libexec/openjdk.jdk"
+    dest: "/Library/Java/JavaVirtualMachines/openjdk-17.jdk"
+    state: link
+  become: true
+
+- shell: /usr/libexec/java_home
+  register: java_home
+
+- name: set global envvars
+  ansible.builtin.lineinfile:
+    path: "~/.zprofile"
+    regexp: "{{ item.regexp }}"
+    line: "{{ item.line }}" 
+    create: yes
+  loop:
+    - regexp: '^JAVA_HOME'
+      line: "JAVA_HOME={{java_home.stdout_lines[0]}}"
+    - regexp: '^CPPFLAGS'
+      line: CPPFLAGS="-I{{ brew_jdk.stdout_lines[0] }}/include"
+
+- name: Upgrade pip
+  ansible.builtin.pip:
+    name:
+      - pip
+    state: latest
+
+- name: Install pip dependencies
+  ansible.builtin.pip:
+    name:
+      - ansible
+      - ansible-runner
+      - wheel
+    state: latest
+
+- name: install ansible-rulebook
+  environment:
+    PIP_NO_BINARY: jpy
+    JAVA_HOME: "{{ java_home.stdout_lines[0] }}"
+  ansible.builtin.pip:
+    name: "git+https://{{ github_token }}@github.com/ansible/ansible-rulebook.git"
```

Comment:
github_token is no longer necessary since the repo is public. I also think its better if the users install the packge from pypi, not the repo?

Reply:
fixed